modules/opportunities/server/controllers/OpportunitiesServerController.ts

```diff
@@ -836,15 +836,11 @@ class OpportunitiesServerController {
 			name: opportunity.name,
 			short: opportunity.short,
 			description: opportunity.description,
-			earn_format_mnoney: CoreServerHelpers.formatMoney(opportunity.opportunityTypeCd === 'sprint-with-us' ? opportunity.phases.aggregate.target : opportunity.earn, 2),
 			earn: CoreServerHelpers.formatMoney(opportunity.opportunityTypeCd === 'sprint-with-us' ? opportunity.phases.aggregate.target : opportunity.earn, 2),
```

Comment:
Is this deleted line related to this fix, or just general cleanup of dead code?

Reply:
Clean-up of dead code